### PR TITLE
Added angularjs tutorial in learn-single-page-app-way.html.md

### PR DESCRIPTION
**Pull request by lemelon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,11 @@
     "npm": "1.3"
   },
   "dependencies": {
+<<<<<<< HEAD
```
Comment (clochix) [inline package.json]: You commited a conflict, could you please clean this ? (and I don't think we need to update docpad).

**Comment by frankrousseau** [comment]
Another remark: the sample code should be written for bookmarks not for books.

**Comment by clochix** [comment]
Thanks a lot, this is really great work. Let's move the sample app from your repository to a branch of cozy-tutorial (for now, we'll update the structure of cozy-tutorial later) and update the links. Then we'll merge this and publish your first contribution ☺

**Comment by clochix** [comment]
@lemelon is not available today, so I merged its sample app into a new branch of cozy tutorial, updated the links into the SPA documentation and merged everything.

**Pull request merged**